### PR TITLE
Add Documentation for Fuel-Subgraph

### PR DESCRIPTION
**Pull request by Salka1988**
Provided detailed instructions and on how to use Fuel-Subgraph

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+
+```bash
+initdb -D .postgres -E UTF8 --locale=C
+pg_ctl -D .postgres -l logfile start
+createdb graph-node
+```
+`Note:` In case of issues, restart the database by deleting it and then starting it again
+
+### 3. Build Firehose-Extract
+Navigate to Firehose-Extract folder and run the following command in the terminal:
```
Comment (Dentosal) [inline README.md:33]:
```suggestion
Navigate to `firehose-extract` folder and run the following command in the terminal:
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+
+```bash
+cargo build
+```
+
+### 4. Start Fuel-Firehose
+Ensure all arguments in `devel/fuelfire/fuelfire.yaml` are uncommented, then run the following command:
```
Comment (Dentosal) [inline README.md:40]: Are these uncommented by default? Why would the user need to ensure this, if they have not modified those?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+
+```bash
+./firehose-fuel/devel/fuelfire/start.sh
+```
+
+### 5. Run Fuel-Graph-Node
+Navigate to Fuel-Graph-Node folder and run the following command in the terminal:
```
Comment (Dentosal) [inline README.md:47]:
```suggestion
Navigate to `fuel-graph-node` folder and run the following command in the terminal:
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,79 @@
...
+
+```bash
+cargo run --bin graph-node -- --config fuel_config.toml --ipfs http://127.0.0.1:5001 --node-id fuel-node-indexer
+```
+
+## Working with Fuel-Subgraphs
+
+### 1. Install and Build Fuel-Graph-Tooling
+Navigate to Fuel-Graph-Tooling folder and run:
```
Comment (Dentosal) [inline README.md:56]:
```suggestion
Navigate to `fuel-graph-tooling` folder and run:
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,79 @@
...
+
+```bash
+pnpm install
+pnpm build
+```
+
+
+### 2. Building and Deploying Fuel-Subgraphs
+Initialize a new subgraph using the appropriate [.yaml](fuel-example-subgraph/subgraph.yaml) file, then navigate to the subgraph folder and execute the following commands:
```
Comment (Dentosal) [inline README.md:65]: How do I initialize the subraph using the yaml file? A link to official instructions or something would help.

**Review by Dentosal — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Salka1988) [reply README.md:40]: It's not standard and you're right. I will delete it.

**Review by Dentosal — APPROVED** [review]

**Pull request merged**